### PR TITLE
Implement VM starter kit solutions for windows and linux

### PR DESCRIPTION
**Pull request by derek-wade**
The solution folders are named:
- vm-starter-kit-linux-v1
- vm-starter-kit-windows-v1

### CODE REVIEW

File: solutions/vm-starter-kit-linux-v1/configuration.metadata.json [hunk 1]
```diff
@@ -1,63 +1,225 @@
...
+        {
+            "id": "base2",
+            "title": "Backup & Monitoring",
+            "subTitle": "Builds on connecting to a Virtual Machine with Backup and Monitoring.",
+            "cost": {
+                "value": 12,
+                "currency": "USD",
+                "type": "absolute",
+                "cadence": "daily",
+                "tooltip": "Example tooltip describing the pricing, customizable."
```
Comment (unrealab) [inline solutions/vm-starter-kit-linux-v1/configuration.metadata.json:130]: same here. Please update tool tips.

File: solutions/vm-starter-kit-windows-v1/configuration.metadata.json [hunk 2]
```diff
@@ -1,63 +1,225 @@
...
+            "title": "Get Connected",
+            "subTitle": "A starter solution to get setup quickly plus remote management",
             "cost": {
-                "value": 150,
+                "value": 11,
                 "currency": "USD",
                 "type": "absolute",
-                "cadence": "monthly",
+                "cadence": "daily",
                 "tooltip": "Example tooltip describing the pricing, customizable."
```
Comment (unrealab) [inline solutions/vm-starter-kit-windows-v1/configuration.metadata.json:52]: please update tooltip

File: solutiongroups/vmStarterKit.json
[the remark this answers is not in the document]
Comment (derek-wade) [reply solutiongroups/vmStarterKit.json:1]: The solution group files are deleted from the PR

File: solutiongroups/defaultLandingVmBrowse2.json
[the remark this answers is not in the document]
Comment (derek-wade) [reply solutiongroups/defaultLandingVmBrowse2.json:1]: The solution group files are deleted from the PR

File: solutions/vm-starter-kit-linux-v1/configuration.metadata.json [hunk 3]
```diff
@@ -1,63 +1,225 @@
...
+            "title": "Get Connected",
+            "subTitle": "A starter solution to get setup quickly plus remote management",
             "cost": {
-                "value": 150,
+                "value": 8,
                 "currency": "USD",
                 "type": "absolute",
-                "cadence": "monthly",
+                "cadence": "daily",
                 "tooltip": "Example tooltip describing the pricing, customizable."
```
[the remark this answers is not in the document]
Comment (derek-wade) [reply solutions/vm-starter-kit-linux-v1/configuration.metadata.json]: Tooltip is now updated.

File: solutions/vm-starter-kit-linux-v1/configuration.metadata.json (hunk 1 above)
Comment (derek-wade) [reply solutions/vm-starter-kit-linux-v1/configuration.metadata.json]: Tooltip is now updated.

File: solutions/vm-starter-kit-windows-v1/configuration.metadata.json (hunk 2 above)
Comment (derek-wade) [reply solutions/vm-starter-kit-windows-v1/configuration.metadata.json]: Tooltip is now updated.

**Review by unrealab — APPROVED** [review]

File: solutions/vm-starter-kit-linux-v1/discovery.metadata.json [hunk 4]
```diff
@@ -1,5 +1,8 @@
 {
-  "icon": null,
-  "title": "Starter template",
-  "shortDescription": "This sample scenario template helps illustrate how the metadata will show up in solutions center"
+  "icon": {
+    "iconType": "AssetIcon",
+    "asset": "Microsoft.Compute/virtualMachines"
+  },
+  "title": "VM Starter Kit Linux",
```
[the remark this answers is not in the document]
Comment (unrealab) [reply solutions/vm-starter-kit-linux-v1/discovery.metadata.json:6]: Let us go with this name for now and update the name later if we need to.

**Pull request merged**